services/app-api/src/domain-models/UserType.d.ts

```diff
@@ -1,29 +1,26 @@
-// CognitoUserType is our type for representing the information we get
-// from Cognito about a given user
-
 type UserType = StateUserType | CMSUserType | AdminUserType
 
 type StateUserType = {
+    id?: string
     role: 'STATE_USER'
     email: string
-    name: string
-    state_code: string
+    stateCode: string
     givenName: string
     familyName: string
 }
 
 type CMSUserType = {
+    id?: string
     role: 'CMS_USER'
     email: string
```

Comment:
🤔 @macrael  Should adding `states` CMSUserType be an add on ticket after this work?  I get that it would require changing a bunch of stuff in tests so seems out of scope here. Does seem like kind of blocker to us actually using the states that are going to be surfaced on the new prisma user type  in the future though...

Reply:
I ended up doing this. I went ahead and added resovlers to call these apis and that required adding stateAssignments so I could actually get that data out of the API